web/public/js/directives/cd-email-users/index.js

```diff
@@ -0,0 +1,70 @@
+;(function () {
+  /*global mailForm*/
+  'use strict';
+
+  angular
+    .module('cpZenPlatform')
+    .component('cdEmailUsers', {
+      bindings: {
+        users: '<',
+        dojoId: '<'
+      },
+      controller: ['usSpinnerService', '$translate', '$rootScope', 'utilsService',
+      'cdDojoService', 'alertService', 'atomicNotifyService', '$scope',
+      function (usSpinnerService, $translate, $rootScope, utilsService,
+        cdDojoService, alertService, atomicNotifyService, $scope) {
+        var ctrl = this;
+        ctrl.editorOptions = utilsService.getCKEditorConfig();
+        var defaultContent = ctrl.content = $translate.instant('Type the content of your email here');
+        ctrl.title = '';
+        cdDojoService.load(ctrl.dojoId)
+        .then(function (dojo) {
+          ctrl.dojo = dojo.data;
+        })
+        .catch(function (err) {
+          alertService.showError($translate.instant('Error while loading Dojo' + JSON.stringify(err)));
+        });
+        ctrl.update = function () {
+          if (ctrl.users) {
+            ctrl.parents = _.uniqBy(_.map(_.pickBy(ctrl.users, 'parent'), 'parent'), 'userId');
+            //TODO: take into account users without emails, should the "To" field change?
+            ctrl.total = ctrl.users.length + ctrl.parents.length;
+          }
+        };
+
+        ctrl.$onChanges = function () {
+          ctrl.update();
+        };
+
+        ctrl.removeUser = function (userId) {
+          ctrl.users = _.reject(ctrl.users, {userId: userId});
+          ctrl.update();
+        };
+
+        ctrl.sendEmail = function () {
+          usSpinnerService.spin('manage-dojo-users-spinner');
+          ctrl.payload = {
+            userIds: _.map(ctrl.users, 'userId'),
+            data: {
+              subject: ctrl.title,
```

Comment:
Let's do subject open, if this causes confusion later we can change.

Footer. Thoughts on this? 
'This email is from {DOJO NAME} Dojo, not from the CoderDojo Foundation. Any content is from an admin of the Dojo and a Dojo should not ask for your password or personal account information.'